prime-angular-frontend/src/app/modules/paper-enrolment/pages/regulatory-page/regulatory-overview.component.ts

```diff
@@ -0,0 +1,83 @@
+import { Component, OnInit, Input } from '@angular/core';
+import { Router, ActivatedRoute } from '@angular/router';
+
+import { RouteUtils } from '@lib/utils/route-utils.class';
+
+import { PaperEnrolmentRoutes } from '@paper-enrolment/paper-enrolment.routes';
+
+@Component({
+  selector: 'app-regulatory-overview',
+  template: `
+    <app-page-section>
+      <app-page-subheader>
+        <ng-container appPageSubheaderTitle>College Licence Information</ng-container>
+
+        <button *ngIf="true"
+                mat-icon-button
+                matTooltip="Edit College Licenses"
+                (click)="onRoute(PaperEnrolmentRoutes.REGULATORY)">
+          <mat-icon>edit</mat-icon>
+        </button>
+      </app-page-subheader>
+
+      <ng-container *ngFor="let certification of certifications; let i = index;">
+
+        <app-enrollee-property title="College Licence"
+                               [makeBold]="true">
+          {{ certification.collegeCode | configCode: 'colleges' | default }}
+        </app-enrollee-property>
+
+        <app-enrollee-property title="Licence Class">
+          {{ certification.licenseCode | configCode: 'licenses' | default }}
+        </app-enrollee-property>
+
+        <app-enrollee-property title="Licence Number">
+          {{ certification.licenseNumber | default }}
+        </app-enrollee-property>
+
+        <app-enrollee-property title="Prescriber ID">
+          {{ certification.practitionerId | default }}
+        </app-enrollee-property>
+
+        <app-enrollee-property title="Renewal Date">
+          {{ certification.renewalDate | formatDate | default }}
+        </app-enrollee-property>
+
+        <app-enrollee-property title="Advanced Practices">
+          {{ certification.practiceCode | configCode: 'practices' | default }}
+        </app-enrollee-property>
+
+      </ng-container>
+
+      <app-enrollee-property *ngIf="!certifications?.length"
+                             title="College Certification"
+                             [makeBold]="true">
+        None
+      </app-enrollee-property>
+    </app-page-section>
+  `,
+  styles: [
+  ]
+})
+export class RegulatoryOverviewComponent implements OnInit {
+
+  @Input() certifications = [];
```

Comment:
Missing access modifier and typing.  Instead of adding defaults to the member variables directly we add them in constructor.  This allows us to add specific typings like `Contact[]` to the member variable instead of just an array and relying on type inference.  As it is now I could pass you anything in the array, and if you add a typing and default you'll get a linting error.  Setting defaults in the constructor provides consistency and every developer knows they can check the constructor.